### DISCUSSION TITLE
Chrome file upload

### DISCUSSION THREAD

**Discussion by cabernetfrancmerlot** [masbug/flysystem-google-drive-ext#106] [Q&A]
Hi,
I use this library with Laravel.
Files upload with Firefox, but with Chrome and Chromium the files don't upload. They are only deleted or downloaded.
I can't find a solution to the problem, what could it be?

Thank

**Comment by parallels999** [comment]
Update your chrome, I use chrome and everything works as expected
surely it is not a problem of this library, but of your environment
